### PR TITLE
Part 2 of the task

### PR DESCRIPTION
**Pull request by DarkmatterVale**
In this PR, I have added numerous features.

1) Much needed Icon. Although it is not much, it is definitely better than the default icon
2) Script editing. This allows the user to edit a script real time & see its changes within HaikuPlot
3) Bug fixes. Multiple fixes in terms of layout & display
4) Added About window. Simple to start with, can be expanded if others would like
5) Using LayoutManager now
6) Added "Export as..." which allows the user to export the plot to any of the file types supported on his/her machine

### CODE REVIEW

**Comment by waddlesplash** [comment]
For the AboutWindow -- How about just using BAboutWindow?

File: HaikuPlot/HaikuPlot.cpp [hunk 1]
```diff
@@ -151,6 +248,11 @@ void HaikuPlot::GeneratePlot(const entry_ref &ref)
 	if (file.InitCheck() != B_OK)
 		return;
 	
+	fScriptView->SetText("");
+	if (BTranslationUtils::GetStyledText(&file, fScriptView) != B_OK)
+	{
+	}
```
Comment (waddlesplash) [inline HaikuPlot/HaikuPlot.cpp]: Maybe add some error handling code here?

File: HaikuPlot/HaikuPlot.cpp [hunk 2]
```diff
@@ -17,42 +18,100 @@
...
+				.Add(fPictureView)
+			.End()
+			.AddGroup(B_VERTICAL, spacing / 2)
+				.Add(fScriptView)
+			.End()
+		.SetInsets(spacing)
+		.View();
+
+	BLayoutBuilder::Group<>(this, B_VERTICAL, 0)
+		.Add(fMenuBar)
```
Comment (waddlesplash) [inline HaikuPlot/HaikuPlot.cpp]: Adding a .SetInsets(0) before the first .Add might make this look nicer.

File: HaikuPlot/HaikuPlot.cpp [hunk 3]
```diff
@@ -17,42 +18,100 @@
...
+	
+	fPictureView = new BView(BRect(0,20,650,600), "picture_view",
+		B_FOLLOW_NONE, B_WILL_DRAW);
+		
+	fScriptView = new BTextView("script_view", B_WILL_DRAW);
+	
+	static const float spacing = be_control_look->DefaultItemSpacing() / 2;
+	fMainSplitView =
+		BLayoutBuilder::Split<>(B_HORIZONTAL)
+			.AddGroup(B_VERTICAL)
```
Comment (waddlesplash) [inline HaikuPlot/HaikuPlot.cpp]: Same here - add a .SetInsets(0)

**Comment by DarkmatterVale** [comment]
GCI task: https://codein.withgoogle.com/dashboard/task-instances/4627343054209024/?sp-page=1

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**